### ISSUE TITLE
Cannot import keylooger

### ISSUE THREAD

**Issue by mzfr** [abhisharma404/vault#95]
In [vault.py#L912](https://github.com/abhisharma404/vault/blob/master/src/vault.py#L912)  we are doing 
```python
 keyloggerObj = keylogger.Keylogger(interval=args.interval,
```
but instead it should be 
```python
 keyloggerObj = keylogger.keylogger(interval=args.interval,
```
Notice the lowercase `k` after ` = keylogger.`

This is because the file name is [utilities/keylogger](https://github.com/abhisharma404/vault/tree/master/src/lib/utilities/keylogger)

**Comment by sap143** [comment]
I think line number should be 928 instead of 912 in vault.py

**Comment by sap143** [comment]
I would like to work on this .

**Comment by mzfr** [comment]
@sap143 Okay You can work on this.

Just remember you won't be allowed to work on any other issue from this projec unless your PR if accepted or you abandon the task.

**Comment by mzfr** [comment]
Fixed by #100
